server/shared/mail/index.js

```diff
@@ -62,9 +64,12 @@ function resetPassword(user, token) {
 }
 
 function sendCommentNotification(users, comment) {
+  const { repositoryId, activityId } = comment;
+  const href = activityUrl(repositoryId.toString(), activityId.toString());
```

Comment:
Consider using interpolation for repositoryId like for activityId so you don't have to do casting.